tests/client.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from datetime import datetime
+
+import requests
+from retrying import retry
+
+from udata_piwik.client import track
+
+from .conftest import PiwikSettings
+
+BASE_URL = 'http://{0.PIWIK_URL}'.format(PiwikSettings)
+RESET_URL = '{0}/reset.php'.format(BASE_URL)
+
+
+@retry(stop_max_attempt_number=20, wait_fixed=500)
```

Comment:
Thanks for this,I was looking for something like that :ok_hand:

Reply:
❤️